### PR TITLE
javascript to embed comments from discourse forum

### PR DESCRIPTION
**Pull request by rriemann**
The comment system can be changed to embed topics from the Discourse
forum software: http://discourse.org/

More details: https://meta.discourse.org/t/embedding-discourse-comments-via-javascript/31963

### CODE REVIEW

File: _includes/comments-providers/discourse.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% if site.comments.discourse.server %}
+{% capture canonical %}{{ site.url }}{% if site.permalink contains '.html' %}{{ page.url }}{% else %}{{ page.url | remove:'index.html' | strip_slash }}{% endif %}{% endcapture %}
```
Comment (mmistakes) [inline _includes/comments-providers/discourse.html]: Might want to add `{{ site.baseurl }}` after `{{ site.url }}` as the theme supports that and if defined in `_config.yml` will properly assign the wrong page url to `canonical`.

**Comment by rriemann** [comment]
Ok, I reuse the base_path snippet.

**Comment by mmistakes** [comment]
@rriemann I've never setup Discourse before. Are the resoruces on https://meta.discourse.org/t/embedding-discourse-comments-via-javascript/31963 all that someone needs to get up and running? I want to be sure I include all the relevant info in the theme's docs because I know I'll get support issues about it eventually :wink:.

**Comment by rriemann** [comment]
Yes, it should be fine. The Discourse meta forum is a great place to figure out 
problems and if there is a problem at some point, the provided link is not 
only the right place to find some advice, but also to ask questions.

**Pull request merged**